lib/event-parser.js

```diff
@@ -2,10 +2,34 @@
 
 const EventParserError = require('./event-parser-error');
 
+const ModelEvent = require('./model-event');
+
 class EventParser {
+	/**
+	 * Get the event actions
+	 *
+	 * @param {String} message object
+	 * @return {Array} Event actions
+	 * @throw EventParserError
+	 */
+	static async getSubscribers(message) {
+		if(!message || typeof message !== 'object')
+			throw new EventParserError('message is required', EventParserError.codes.INVALID_DATA);
+
+		const { client = 'core', entity, event } = message;
+
+		if(!entity || !event)
+			throw new EventParserError('message entity and event are required', EventParserError.codes.INVALID_DATA);
+
+		const modelEvent = new ModelEvent();
+		const getEvent = await modelEvent.getEvent(client, entity, event);
```

Comment:
```suggestion
		const events = await modelEvent.getEvent(client, entity, event);
```

Por otro lado, ya podría devolver la posición `[0]` (o `false`), teniendo en cuenta que se llama `getEvent()` y no `getEvents()`.

Reply:
Se realizó el cambio por { subscriptions } ya que se devuelve el evento desde el modelo